specs/core/0_beacon-chain.md

```diff
@@ -520,6 +518,25 @@ def get_beacon_proposer(state:BeaconState, slot: int) -> ValidatorRecord:
     return state.validators[index]
 ```
 
+The following is a function that determines the validators that participated in an attestation:
+
+```python
+def get_attestation_participants(state: State,
```

Comment:
Should this update be part of #145 instead of this PR? If not, could you explain more about why we need two bits for having epochs?